### ISSUE TITLE
ERROR: VisualServerRaster::_draw_viewport when 2d/3d editor viewport been covered

### ISSUE THREAD

**Issue by marynate** [godotengine/godot#585]
error message:

```
ERROR: VisualServerRaster::_draw_viewport:  Condition ' desired_rect.width<=0 ||
 desired_rect.height<=0 ' is true.
At:  servers\visual\visual_server_raster.cpp:5763
```

how to reproduce:

```
1. run scene from editor
2. cover 2d or 3d viewport by pulling up "output" pannel
3. repeating error message printing in console
```

**Comment by akien-mga** [comment]
I confirm this is still valid as of bd736e5.

**Comment by akien-mga** [comment]
Is this still valid with the refactored panels in 2.0?

**Comment by bojidar-bg** [comment]
@akien-mga ~ Unfortunately, I'm unable to resize the panel that high up.

**Comment by akien-mga** [comment]
Closing then, we'll reopen or open a new issue if it shows up again.

**Comment by bojidar-bg** [comment]
@akien-mga ~ I was able to actually reproduce it with a different panel, should I reopen?

**Comment by bojidar-bg** [comment]
Ok, to add details, this is reproducible with the tilemap pane, when dragging it all the way to the right on 7913e792acd656469b29fb90be1dbb7c06a855ba (or a few commits earlier)

**Comment by akien-mga** [comment]
@bojidar-bg Can you still reproduce it in master?

**Comment by bojidar-bg** [comment]
No, I can no longer reproduce the exact error. Now I get another error though: :smiley: 
```
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL       Type: Error     ID: 35  Severity:
     High  Message: GL_INVALID_OPERATION in glBindTexture(non-gen name)
   At: drivers/gles3/rasterizer_gles3.cpp:122.
```

**Comment by flashyincceo** [comment]
Is this still an issue? I am not able to reproduce the problem by dragging the tilemap pane or the output panel.

**Comment by akien-mga** [comment]
Assuming fixed in 3.0. Please comment if you can still reproduce the same issue, or open a new issue if the error message differs.
